ssl/ssl_rsa.c

```diff
@@ -1009,8 +1009,9 @@ int SSL_CTX_use_serverinfo_file(SSL_CTX *ctx, const char *file)
         extension = NULL;
     }
 
-    ret = SSL_CTX_use_serverinfo_ex(ctx, version, serverinfo,
-                                    serverinfo_length);
+    if (version != 0)
```

Comment:
Yup, it's sufficient...  but that's not entirely obvious, and I'd say that there's reason to have a `OPENSSL_assert(version != 0);` around here, to make sure we catch a possible future goof-up in the `for` loop above.